test/SourceKit/DocumentStructure/structure.swift.empty.response

```diff
@@ -1177,7 +1177,40 @@
       key.nameoffset: 1796,
       key.namelength: 1,
       key.bodyoffset: 1799,
-      key.bodylength: 109
+      key.bodylength: 109,
+      key.substructure: [
+        {
+          key.kind: source.lang.swift.decl.function.method.instance,
+          key.accessibility: source.lang.swift.accessibility.internal,
+          key.name: "init(x:y:)",
```

Comment:
Hm, this seems suspect. An initializer should never be treated as a function!

Reply:
As far as I understand, this is not an initializer.
https://github.com/apple/swift/blob/d7c0a61c9334ae66adf8d48cf87e9441e4d9218f/test/SourceKit/DocumentStructure/Inputs/main.swift#L114-L117

Currently this input produces the "expected declaration" error.

Reply:
Ahhh, whoops! Okay, then this is behaving correctly, though I wonder if SourceKit should be escaping it.

Reply:
As far as I can see from the other declarations with backticks, it shouldn't.

Declaration:

https://github.com/apple/swift/blob/7ba9d655dfeff6e6dde58c6ab05453d096f0b58a/test/SourceKit/DocumentStructure/Inputs/main.swift#L120

SourceKit output:

https://github.com/apple/swift/blob/7ba9d655dfeff6e6dde58c6ab05453d096f0b58a/test/SourceKit/DocumentStructure/structure.swift.response#L1193-L1196

Reply:
Yeah, but `foo` isn't a reserved word normally. `` func `func` ``, for example, might be more interesting.

Anyway, this isn't changed by your work here, so we don't need to do something about it now.